src/set_theory/cardinal.lean

```diff
@@ -275,6 +275,10 @@ instance : canonically_ordered_comm_semiring cardinal.{u} :=
   ..cardinal.order_bot,
   ..cardinal.comm_semiring, ..cardinal.linear_order }
 
+noncomputable instance : canonically_linear_ordered_add_monoid cardinal.{u} :=
+{ .. (infer_instance : canonically_ordered_add_monoid cardinal.{u}),
+  .. cardinal.linear_order }
+
```

Comment:
This one has to be noncomputable because `cardinal.linear_order` contains decidable instances which are themselves noncomputable. The typeclasses above this one do not suffer from that problem, because they throw away those fields